### DISCUSSION TITLE
Error-13 during installation

### DISCUSSION THREAD

**Discussion by ghost** [lukaspieper/Gcam-Services-Provider#96] [Q&A]
Hi there,

I had installed the "Gcam Service Provider (basic)" in the past on my Shiftphone. The first installation in the past went well.
But I uninstalled the service after some days and I though that I uninstalled it correctly :-).

Now I wanna install the service again over the FDroid App Store to use it with the GCam photo app.
But I get this error message:

![Screenshot_20221213-100928_F-Droid](https://user-images.githubusercontent.com/67652981/207276579-d70c0443-6fcb-48ac-95c3-18c63ae527c8.png)

Maybe you could help me with that issue, that I can solve this.
Best regards, Hannes

**Comment by lukaspieper** [comment]
Hi Hannes,

thanks for reaching out.

> Now I wanna install the service again over the FDroid App Store to use it with the GCam photo app.

What do you mean with "GCam photo app"? From the screenshot you are installing the basic version that does not support the button opening your gallery without Google Photos. 

> Shiftphone

I never heard of that phone (quickly looked it up but haven't found too much information about the OS). What OS is it running, does it come with Google Services or microg (than you could not have my app installed previously)? I assume you already checked the app list inside settings (my app has no icon on the app drawer)?

The message in your screenshot says that there is an app on your device providing the same service my app would "fake". Do you know how to use ADB? It's possible to find the incompatible app with that tool.

**Comment by ghost** [comment]
Thank you lukas for your reply!

This is a Shiftphone : [Shiftphone](https://www.shiftphones.com/en/)
It has a google free OS called Shift OS L and I run microG on it to simulate the google services.

Some weeks ago I followed that shiftphone forum comments to install: [NGcam](https://www.celsoazevedo.com/files/android/google-camera/dev-Nikita/f/dl13/). 
Here is the instruction of the Shiftphone forum: [Shiftphone forum](https://forum.shiftphones.com/threads/welche-alternative-kamera-app-thirdparty-camera-app-koennt-ihr-empfehlen.3517/post-42781)

Sadly I am not fine with ADB. But it seams to be a conflict with my installed microG, right? But I am pretty sure I followed the instructions of the Shiftphone forum. And I also had microG installed and it worked well.

Do you have a screenshot, where I could find your app in the android settings? Then I will check this out, if there is one installation left.

Regards Hannes

**Reply by lukaspieper** [reply]
I read a little thru that forum. Here are the following possibilities:

1. When using microg you cannot (and don't need) use either version of my app. To get the gallery button working you can use the "GCam Photo Preview by Calyx" as mentioned on that forum.
2. With no installed microg (or Google Services) it is possible to use both versions of my app. The photos version offers support for the gallery button without further steps.

In your case with microg, Gcam should just work (except for that button). Having microg installed also means that you don't have any old version of my app installed because that's technically not possible (for the same reasons you cannot install my app thru F-Droid).

Hope this helped!

**Comment by ghost** [comment]
Hey Lukas,

yes this is helpful. Thank you so much.

Regards Hannes
